### DISCUSSION TITLE
Manually change position in taskbar view

### DISCUSSION THREAD

**Discussion by ilitur** [zhongyang219/TrafficMonitor#1649] [General]
Hi,

Is possible to change the position (specially the vertical position) for the taskbar view? In my Windows 11 Version (22H2) the info are shown in the top of the taskbar and it's a lot unesthetic. I'll prefere to vergically align in the middle of the taskbar.

![screen](https://github.com/zhongyang219/TrafficMonitor/assets/37334704/247f4116-c90d-48dd-b92b-6aeb378abe13)

**Comment by Peach1200** [comment]
The same happens on my laptop too. 
I'm using a a Windows 11 pc, 2880x1800px display with 175% zoom.
Having the chance to manually align the taskbar window would be great!

![Immagine 2023-11-16 141508](https://github.com/zhongyang219/TrafficMonitor/assets/57133234/514fea85-67af-44b5-b0a7-a917d617fb06)

**Comment by markcytai** [comment]
same problem here! Windows 11 Pro 22h2,  2880x1800px display with 200% zoom.
running Translucent TB and Mica for Everyone

![trafficmon_issue](https://github.com/zhongyang219/TrafficMonitor/assets/141721717/42ddc5af-bcff-450d-b7cf-d5a064a90ab7)

**Comment by Wangzx5** [comment]
![image](https://github.com/zhongyang219/TrafficMonitor/assets/43327345/d15ff001-742d-4a16-b9e5-1647ca9d0d8d)
同样的问题，，有什么解决办法么

**Comment by seethahere** [comment]
Same issue here. 

Do we have a fix yet? (Feb 2024)

![Screenshot 2024-02-22 190801](https://github.com/zhongyang219/TrafficMonitor/assets/79074772/7635f09f-908d-49d9-aa57-6da56018dc08)

**Comment by ahm-asuh** [comment]
same issue....

**Comment by ahm-asuh** [comment]
https://github.com/liukyle/TrafficMonitor/releases

yes.. fix, thank brother liukyle
